### ISSUE TITLE
Profile INI files Not being used.

### ISSUE THREAD

**Issue by rdunlap** [TanninOne/modorganizer#681]
It seems the profile INI files are not yet being used by the game. I have 2 profiles, 1 for having 1920X1080 windowed when i am developing a mod, and 1 at 5670X1080 full screen when playing and switching profiles has no effect on the resolution meaning at least the falloutprefs.ini is not being used. Changing a setting in the Games default location ini file however shows the change in either profile.

Robert

**Comment by KeltecRFB** [comment]
#686 Check your folder path, this may actually be the problem.  I thought it was this too but it ended up being the path configuration in settings (cross screw driver and wrench icon) > paths

**Comment by rdunlap** [comment]
All my paths are correct. I am not using the multigame installation I am using a dedicated installation and its in a NON traditional path as well. I suspect there is some issue with permissions to special folders coming into this as well. 

Robert

**Comment by KeltecRFB** [comment]
Running as admin?

**Comment by rdunlap** [comment]
Yes. I even tried disabling my AV since it has in the past not played nice with the alpha but it just does not use them for some reason. My game is installed on an SSD *(E drive) and MO is a sub of the game directory. I dunno.

**Comment by KeltecRFB** [comment]
The INI seems to be now working for me, but now certain mods are not installing.

**Comment by KeltecRFB** [comment]
Hmm, I changed the load order of the mods and the INI did not load.

**Comment by TwistedByte** [comment]
It seems that mo don't load any ini from profile folder on standalone installation not even it's own 
initweaks.ini. I have symlinked ini's from profile with the ones from "my games\fallout4" and in 
fallout4.ini i must ad to archive section for the mods to work 
bInvalidateOlderFiles=1
sResourceDataDirsFinal=
(because it cant read bInvalidateOlderFiles=1 from initweaks,ini in mo profiles)

**Comment by ogrotten** [comment]
@TwistedByte Did your tweak fix/workaround the problem?

If so, what steps did you take to make it work? I'm wondering if this is the underlying problem that causes #684 and would like to see if your steps fix that.

**Comment by rdunlap** [comment]
not for me. I get no load at all of the profile INI's. I even get the crash on profile change like you do so its something not working in the UI i suspect.

Robert

**Comment by haileyambition** [comment]
Just tried @TwistedByte method.  Does seem to work for me either.

**Comment by wmcook32** [comment]
The issue as it seems to me is the simple fact the virtualization of the directories are not fully working as intended. work arounds might work or might not. I am not a programmer in the sense of Tannin but a web programmer. So it is a completely different language and possibly tools.

**Comment by rdunlap** [comment]
This issue is unchanged in Beta 2. 

Sorry about the duplicated report. can only claim tired brain yesterday as I had been up for nearly a day and a half obsessing over a problem with a mod.

Robert

**Comment by rdunlap** [comment]
This issue is unchanged in Beta 3

Robert
